xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/definitions.h

```diff
@@ -175,9 +176,39 @@ typedef struct AddonToKodiFuncTable_kodi_gui_window
   bool (*show)(void* kodiBase, void* handle);
   bool (*close)(void* kodiBase, void* handle);
   bool (*do_modal)(void* kodiBase, void* handle);
+
+  /* Window control calls */
+  bool (*set_focus_id)(void* kodiBase, void* handle, int control_id);
+  int (*get_focus_id)(void* kodiBase, void* handle);
+  void (*set_control_label)(void* kodiBase, void* handle, int control_id, const char* label);
+
+  /* Window property calls */
+  void (*set_property)(void* kodiBase, void* handle, const char* key, const char* value);
+  void (*set_property_int)(void* kodiBase, void* handle, const char* key, int value);
+  void (*set_property_bool)(void* kodiBase, void* handle, const char* key, bool value);
+  void (*set_property_double)(void* kodiBase, void* handle, const char* key, double value);
+  char* (*get_property)(void* kodiBase, void* handle, const char* key);
+  int (*get_property_int)(void* kodiBase, void* handle, const char* key);
+  bool (*get_property_bool)(void* kodiBase, void* handle, const char* key);
+  double (*get_property_double)(void* kodiBase, void* handle, const char* key);
+  void (*clear_properties)(void* kodiBase, void* handle);
+  void (*clear_property)(void* kodiBase, void* handle, const char* key);
+
+  /* List item calls */
   void (*clear_item_list)(void* kodiBase, void* handle);
-  void (*add_list_item)(void* kodiBase, void* handle, void* item, int item_position);
-  void* (*get_list_item)(void* kodiBase, void* handle, int listPos);
+  void (*add_list_item)(void* kodiBase, void* handle, void* item, int list_position);
+  void (*remove_list_item_from_position)(void* kodiBase, void* handle, int list_position);
+  void (*remove_list_item)(void* kodiBase, void* handle, void* item);
+  void* (*get_list_item)(void* kodiBase, void* handle, int list_position);
+  void (*set_current_list_position)(void* kodiBase, void* handle, int list_position);
+  int (*get_current_list_position)(void* kodiBase, void* handle);
+  int (*get_list_size)(void* kodiBase, void* handle);
+  void (*set_container_property)(void* kodiBase, void* handle, const char* key, const char* value);
+  void (*set_container_content)(void* kodiBase, void* handle, const char* value);
+  int (*get_current_container_id)(void* kodiBase, void* handle);
+
+  /* Various calls */
```

Comment:
minor nitpicking: these are not calls, but functions. Seen multiple times in this source file.